package.json

```diff
@@ -17,5 +17,8 @@
     "path": "^0.12.7",
     "traverse": "^0.6.6",
     "yargs": "^14.2.0"
+  },
+  "devDependencies": {
+    "fetch": "^1.1.0"
```

Comment:
@ldragoon  Do we need this dependancy anymore?

Reply:
Looks like it's no longer in use, I removed it.